tests/main/dbus-activation-session/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: D-Bus session services support activation via systemd user session
+
+systems:
+    # Ubuntu Core does not contain the D-Bus user session systemd units
+    - -ubuntu-core-*
+    # Ubuntu 14.04 does not provide user@.service
+    - -ubuntu-14.04-*
+    # CentOS/Amazon Linux give error "Unit user@0.service not loaded."
+    - -amazon-linux-2-*
+    - -centos-7-*
+
+prepare: |
+    snap set system experimental.user-daemons=true
+    snap set system experimental.dbus-activation=true
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+    distro_install_package dbus-user-session
```

Comment:
This is now preinstalled in all systems. I think you can drop it.
In addition, the restore side is missing.